### ISSUE TITLE
not working nowadays

### ISSUE THREAD

**Issue by roccomuso** [o2platform/node-ssl-strip#2]
![screenshot from 2016-12-06 21-08-04](https://cloud.githubusercontent.com/assets/6730217/20941800/7bba56d6-bbf8-11e6-8cd0-24d226da8d0e.png)

**Comment by DinisCruz** [comment]
what was the set-up you had (what browser did you use?)

also , it looks like you tried to open an https page and the proxy is currently only proxying http requests

**Comment by roccomuso** [comment]
Yeah but the ssl strip is meant to "strip" the pages with HTTPS right? that's supposed to be his main goal.

**Comment by DinisCruz** [comment]
yes and that is done with this line https://github.com/o2platform/node-ssl-strip/blob/master/poc/google.coffee#L21 on http traffic

![image](https://cloud.githubusercontent.com/assets/656739/21128992/ed8072b6-c0f6-11e6-86db-9332a9398d01.png)

Note that there is a white list of domains to apply modifications to 

![image](https://cloud.githubusercontent.com/assets/656739/21129007/003306bc-c0f7-11e6-80d1-f1d176f250a2.png)

if you run the poc (i.e. ```./node_modules/.bin/coffee poc/google.coffee```) you are only intercepting google.co.uk and google.com

**Comment by roccomuso** [comment]
alright, but what about [HSTS](https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security)?

**Comment by DinisCruz** [comment]
well HSTS will prevent this from happening.

When you have HSTS you can't really man-in-the-middle like this (which is the point of HSTS :)   )

**Comment by roccomuso** [comment]
That's why it's not working with google.com either.
